internal/test/resources.go

```diff
@@ -560,7 +560,7 @@ func (r *TestResources) NewCryostatWithDiscoveryPortConfig() *model.CryostatInst
 
 func (r *TestResources) NewCryostatWithBuiltInPortConfigDisabled() *model.CryostatInstance {
 	cr := r.NewCryostat()
-	cr.Spec.TargetDiscoveryOptions = &operatorv1beta1.TargetDiscoveryOptions{
+	cr.Spec.TargetDiscoveryOptions = &operatorv1beta2.TargetDiscoveryOptions{
 		DisableBuiltInPortNames:   true,
 		DisableBuiltInPortNumbers: true,
 	}
```

Comment:
Could you add a v1beta1 copy of these CRs (with the suffix V1Beta1) to test/conversion.go, and then add test cases in api/v1beta1/cryostat_conversion_test.go like this:
```go
		Entry("discovery port custom config", (*test.TestResources).NewCryostatWithDiscoveryPortConfigV1Beta1,
			(*test.TestResources).NewCryostatWithDiscoveryPortConfig),
		Entry("discovery port config disabled", (*test.TestResources).NewCryostatWithBuiltInPortConfigDisabledV1Beta1,
			(*test.TestResources).NewCryostatWithBuiltInPortConfigDisabled),
``` 
This will add test cases for the conversion webhook using those CRs.